#
#
# patch "libpurple/protocols/oscar/oscar.c"
#  from [b448ff8a14ab7db9b3ad6ce2c8f17cac8159f702]
#    to [6b8b45623896204448ac4b212a0c631655822ad1]
#
============================================================
--- libpurple/protocols/oscar/oscar.c	b448ff8a14ab7db9b3ad6ce2c8f17cac8159f702
+++ libpurple/protocols/oscar/oscar.c	6b8b45623896204448ac4b212a0c631655822ad1
@@ -4651,7 +4651,8 @@ oscar_send_im(PurpleConnection *gc, cons
 			tmp2 = purple_markup_strip_html(tmp1);
 			is_html = FALSE;
 		} else {
-			tmp2 = g_strdup(tmp1);
+			/* ICQ 6 wants its HTML wrapped in these tags. Oblige it. */
+			tmp2 = g_strdup_printf("<HTML><BODY>%s</BODY></HTML>", tmp1);
 			is_html = TRUE;
 		}
 		g_free(tmp1);
